test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -18,6 +18,273 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
     public class TagHelperParseTreeRewriterTest : TagHelperRewritingTestBase
     {
+        public static TheoryData InvalidHtmlScriptBlockData
+        {
+            get
+            {
+                var factory = CreateDefaultSpanFactory();
+                var blockFactory = new BlockFactory(factory);
+
+                return new TheoryData<string, MarkupBlock>
+                {
+                    {
+                        "<script type><input /></script>",
```

Comment:
I'd be very surprised if any browser accepted this as valid. Neither "type" or the empty string is a [valid MIME type](http://www.w3.org/TR/html5/infrastructure.html#valid-mime-type). And minimized attributes only works for HTML [Boolean attributes](http://www.w3.org/TR/html5/infrastructure.html#boolean-attribute).